packages/react-reconciler/src/ReactFiberRoot.js

```diff
@@ -28,6 +28,13 @@ export type FiberRoot = {
   pendingChildren: any,
   // The currently active root fiber. This is the mutable root of the tree.
   current: Fiber,
+
+  earliestPendingTime: ExpirationTime,
+  latestPendingTime: ExpirationTime,
```

Comment:
I don't get why we need to keep track of these two pending times.

Why isn't earliestPendingTime just `root.current.expirationTime` and latestPendingTime just `earliestSuspendedTime - 1`? Effectively.

Reply:
The pending times are levels we haven’t tried yet. If we try them and they suspend, they are cleared. In other words, need to distinguish between “incomplete” and “incomplete and suspended.”